src/main/java/com/gregtechceu/gtceu/common/machine/multiblock/electric/research/HPCAMachine.java

```diff
@@ -0,0 +1,727 @@
+                        hpcaHandler.cachedCWUt + " / " + hpcaHandler.getMaxCWUt() + " CWU/t")
+                        .withStyle(ChatFormatting.AQUA);
+                    tl.add(Component.translatable(
+                        "gtceu.multiblock.hpca.computation",
+                        cwutInfo).withStyle(ChatFormatting.GRAY));
+                }
+            })
+            .addWorkingStatusLine();
+    }
+
+    private ChatFormatting getDisplayTemperatureColor() {
+        if (temperature < 500) {
+            return ChatFormatting.GREEN;
+        } else if (temperature < 750) {
+            return ChatFormatting.YELLOW;
+        }
+        return ChatFormatting.RED;
+    }
+
+    /*
+    @Override
+    protected void addWarningText(List<Component> textList) {
+        MultiblockDisplayText.builder(textList, isFormed(), false)
+            .addLowPowerLine(hasNotEnoughEnergy)
+            .addCustom(tl -> {
+                if (isStructureFormed()) {
+                    if (temperature > 500) {
+                        // Temperature warning
+                        tl.add(TextComponentUtil.translationWithColor(
+                            TextFormatting.YELLOW,
+                            "gtceu.multiblock.hpca.warning_temperature"));
+
+                        // Active cooler overdrive warning
+                        tl.add(TextComponentUtil.translationWithColor(
+                            TextFormatting.GRAY,
+                            "gtceu.multiblock.hpca.warning_temperature_active_cool"));
+                    }
+
+                    // Structure warnings
+                    hpcaHandler.addWarnings(tl);
+                }
+            })
+            .addMaintenanceProblemLines(getMaintenanceProblems());
+    }
+
+    @Override
+    protected void addErrorText(List<Component> textList) {
+        super.addErrorText(textList);
+        if (isFormed()) {
+            if (temperature > 1000) {
+                textList.add(Component.translatable("gtceu.multiblock.hpca.error_temperature").withStyle(ChatFormatting.RED));
+            }
+            hpcaHandler.addErrors(textList);
+        }
+    }
+
+    @Override
+    public void addBarHoverText(List<Component> hoverList, int index) {
+        if (index == 0) {
+            Component cwutInfo = Component.literal(
+                hpcaHandler.cachedCWUt + " / " + hpcaHandler.getMaxCWUt() + " CWU/t").withStyle(ChatFormatting.AQUA);
+            hoverList.add(Component.translatable(
+                "gtceu.multiblock.hpca.computation",
+                cwutInfo).withStyle(ChatFormatting.GRAY));
+        } else {
+            Component tempInfo = Component.literal(,
+                Math.round(temperature / 10.0D) + "°C").withStyle(getDisplayTemperatureColor());
+            hoverList.add(TextComponentUtil.translationWithColor(
+                TextFormatting.GRAY,
+                "gtceu.multiblock.hpca.temperature",
+                tempInfo));
+        }
+    }
+    */
```

Comment:
I'd recommend just removing this if no longer necessary

Reply:
I want to add those back at some point.